frontend/public/components/catalog.jsx

```diff
@@ -111,10 +135,10 @@ class CatalogListPage extends React.Component {
 
   render() {
     const {loaded, loadError} = this.props;
-    const {items} = this.state;
+    const {items, filters} = this.state;
 
     return <StatusBox data={items} loaded={loaded} loadError={loadError} label="Resources">
-      <CatalogTileViewPage items={items} />
+      <CatalogTileViewPage items={items} filters={filters} />
```

Comment:
I would just define the filters directly in `CatalogTileViewPage` if they're not needed in the parent component. The filter values are already tightly coupled to `CatalogTileViewPage`, and this also avoids copying props to state.

Reply:
somewhat agree, but I do need to pass the counts by type.  The number of imageStream and serviceClass items are easily gathered in the parent class.  I could just pass the counts down and attach to filter obj in child.  Or, I could calculate the count by type in the child, but all items are mixed at this point, seemed easier to do in parent.

Reply:
decided to pass `counts` from parent to child, and have filter state just in child.